pkg/cluster/k8sres.go

```diff
@@ -1253,6 +1257,34 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 	return result
 }
 
+func (c *Cluster) generateStandbyEnvironment(description *acidv1.StandbyDescription) []v1.EnvVar {
+	result := make([]v1.EnvVar, 0)
+
+	if description.S3WalPath == "" {
```

Comment:
does it mean standby that bypasses an S3 bucket with archives and streams directly from the source cluster ?

Reply:
well yes that is the intention.

Reply:
1. that behavior should be documented then
2. is it really the case ? looks like if `S3WalPath` evaluates to the empty string the standby env is empty